server/src/test/java/org/elasticsearch/cluster/coordination/LeaderCheckerTests.java

```diff
@@ -557,6 +557,11 @@ private static class CapturingTransportResponseHandler implements TransportRespo
 
         TransportException transportException;
         boolean successfulResponseReceived;
+        final ThreadPool threadPool;
```

Comment:
Alternatively could save an Executor reference, since it looks like that's all the threadPool is needed for.

Reply:
++ good point, see 27ea9b9eccb.